### PR TITLE
Add doc for HighFive::Attribute

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Review by matz-e — COMMENTED** [review]
LGTM, but needs a merge with master?

File: include/highfive/H5Attribute.hpp [hunk 1]
```diff
@@ -41,89 +41,180 @@ Attribute make_attribute(hid_t hid);
...
+///
+/// Create Attribute with AnnotateTraits::createAttribute
+///
+/// Access Attribute with AnnotateTraits::getAttribute
+///
+/// List all Attribute with AnnotateTraits::listAttributeNames
+///
+/// Check if an Attribute exists with AnnotateTraits::hasAttribute
+///
+/// Delete Attribute with AnnotateTraits::deleteAttribute
```
Comment (alkino) [inline include/highfive/H5Attribute.hpp:54]: Should be replaced by `\sa`?

Comment (1uc) [reply include/highfive/H5Attribute.hpp:54]: Yes, please.

**Comment by alkino** [comment]
Way better thanks!

**Review by 1uc — APPROVED** [review]

**Pull request merged**